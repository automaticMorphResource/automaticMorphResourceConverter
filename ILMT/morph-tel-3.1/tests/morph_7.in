1	cEnAlo	unk	
1	cakkeVra	unk	
1	vyAXi	unk	
1	valana	unk	
1	saMkramiMce	unk	
1	samasyala	unk	
1	pariRkArAniki	unk	
1	wayAru	unk	
1	cese	unk	
1	maMxulalo	unk	
1	gummadini	unk	
1	vAduwunnAru	unk	
1	.	unk	
