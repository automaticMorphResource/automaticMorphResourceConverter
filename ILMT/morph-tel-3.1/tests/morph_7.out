1	cEnAlo	unk	<fs af='cEnA,n,,pl,,,0,0_o'>|<fs af='cEnA,n,,sg,,,lo,lo'>
2	cakkeVra	unk	<fs af='cakkeVra,n,,sg,,d,0,0'>|<fs af='cakkeVra,n,,sg,,o,ti,ti'>
3	vyAXi	unk	<fs af='vyAXi,n,,sg,,d,0,0'>|<fs af='vyAXi,n,,sg,,o,ti,ti'>
4	valana	unk	<fs af='valana,avy,,,,,0,0_avy'>|<fs af='vala,n,,sg,,,na,na'>
5	saMkramiMce	unk	<fs af='saMkramiMcu,v,any,any,any,,e,e'>|<fs af='saMkramiMcu,v,any,any,any,,i,i_e'>|<fs af='saMkramiMcu,v,any,any,any,,e,e'>|<fs af='saMkramiMcu,v,any,any,any,,i,i_e'>
6	samasyala	unk	<fs af='samasya,n,,pl,,o,ti,ti'>
7	pariRkArAniki	unk	<fs af='pariRkAraM,n,,sg,,,ki,ki'>
8	wayAru	unk	<fs af='wayAru,n,,sg,,d,0,0'>|<fs af='wayAru,n,,sg,,o,ti,ti'>
9	cese	unk	<fs af='ceVyyi,v,any,any,any,,e,e'>|<fs af='ceVyyi,v,any,any,any,,i,i_e'>
10	maMxulalo	unk	<fs af='maMxu,n,,pl,,,lo,lo'>
11	gummadini	unk	<fs af='gummadi,n,,sg,,,nu,nu'>|<fs af='gummadi,n,,sg,,,ni,ni'>
12	vAduwunnAru	unk	<fs af='vAdu,v,fm,pl,3,,wunn,wunn'>
13	.	unk	<fs af='&dot;,punc,,,,,,'>
